additional-skills/git/README.md

```diff
@@ -0,0 +1,90 @@
+# Git + GitHub
+
+All sections are required.
+
+## Initializing 
+
+* Demonstrate initializing a new git repository
+* Demonstrate creating an empty root commit
+* Can explain the reasoning for creating an empty root commit
+
+## Basics
+
+* Demonstrate usage of `git status`
+* Demonstrate staging files for commit
+* Demonstrate committing staged files
+* Demonstrate committing a subset of changed files using staged and unstaged files
```

Comment:
How about staging only a subset of a file?

Reply:
Yeah I don’t do this